File: result/translate/starchat/java2python/python/NTH_MULTIPLE_NUMBER_FIBONACCI_SERIES.py
def f_gold(k, n):
        f1, f2, f3 = 0, 1, 0
        for i in range(2, 10000):
            f3 = f1 + f2
            f1 = f2
            f2 = f3
            if f2 % k == 0:
                return n * i
        return 0

File: result/translate/starchat/python2java/java/NTH_MULTIPLE_NUMBER_FIBONACCI_SERIES.java
public static int f_gold( int k, int n ) {
        int f1 = 0, f2 = 1, i = 2;
        while ( i!= 0 ) {
            int f3 = f1 + f2;
            f1 = f2;
            f2 = f3;
            if ( f2 % k == 0 ) return n * i;
            i++;
        }
        return 0;
    }